code/modules/vehicles/wheelchair.dm

```diff
@@ -44,7 +44,7 @@
 		var/datum/component/riding/D = GetComponent(/datum/component/riding)
 		//1.5 (movespeed as of this change) multiplied by 6.7 gets ABOUT 10 (rounded), the old constant for the wheelchair that gets divided by how many arms they have
 		//if that made no sense this simply makes the wheelchair speed change along with movement speed delay
-		D.vehicle_move_delay = round(CONFIG_GET(number/movedelay/run_delay) * 6.7) / min(user.get_num_arms(), 2)
+		D.vehicle_move_delay = round(CONFIG_GET(number/movedelay/run_delay) * 4.1) / min(user.get_num_arms(), 2)
```

Comment:
```suggestion
		D.vehicle_move_delay = round(CONFIG_GET(number/movedelay/run_delay) * 4) / min(user.get_num_arms(), 2)
```
triggers me to an existancial level

Reply:
I made it a bit higher because I'm not exactly sure how round() works but this was my intended purpose

Reply:
round rounds according to normal arithmetic rules, IE 0.5 goes up, 0.499999999999999 goes down. So if it evaluates to anything above 2.499999... you'll round up to 3